tests/src/Context/MinkContext.php

```diff
@@ -150,4 +150,37 @@ public function maximizeBrowserWindow() {
     $this->getSession()->getDriver()->maximizeWindow();
   }
 
+  /**
+   * Set all permissions to admin role.
+   *
+   * @Given I update the administrator role permissions
+   */
+  public function iGiveAllPermToAdminRole() {
+    if ($rid = variable_get('user_admin_role', 0)) {
+      $perms = array();
+
+      foreach (module_implements('permission') as $module) {
+        foreach (module_invoke($module, 'permission') as $key => $perm) {
+          $perms[$key] = $module;
+        }
+      }
+
+      if ($perms) {
+        foreach ($perms as $perm => $module) {
+          $query = db_merge('role_permission');
+          $query->key(array(
+            'rid' => $rid,
+            'permission' => $perm,
+          ));
+          $query->fields(array(
+            'rid' => $rid,
+            'permission' => $perm,
+            'module' => $module,
+          ));
+          $query->execute();
+        }
+      }
+    }
+  }
+
```

Comment:
No, I had to do it because administrator role doesn't inherit all the permissions for dynamically created entities like beans types.